### ISSUE TITLE
Inheriting AMD/UMD classes when using AMD

### ISSUE THREAD

**Issue by viveleroi** [rollup/rollup-plugin-commonjs#150]
Repo: https://github.com/viveleroi/rollup-amd-inherit-bug

I have a class which extends EventEmitter2, and they distribute umd. When our bundle is used with requirejs/AMD, the `createCommonjsModule` doesn't return anything.

Our source can simply be this:

```js
import EventEmitter2 from 'eventemitter2';

export default class Test extends EventEmitter2 {}
```

In EventEmitter2 code, and in the bundled output, we get:

```js
if (typeof define === 'function' && define.amd) {
     // AMD. Register as an anonymous module.
    define(function() {
      return EventEmitter;
    });
  }
...
```

Problem is, the `createCommonjsModule` doesn't handle this. The output assigns the returned value of that method, but in this case, it's `undefined`.

```js
var eventemitter2 = createCommonjsModule(...
// => undefined
```

So when `eventemitter2` is passed to the `inherits` call (`inherits(Test, _EventEmitter);`) it throws an error `Super expression must either be null or a function, not object`

**Comment by Rich-Harris** [comment]
Good news! This is fixed in 6.0.0, via #144 – just tested the repro and if you update this plugin it should all work.

**Comment by viveleroi** [comment]
I missed that was a duplicate, my bad. Glad to hear, I'll try it out.
